### DISCUSSION TITLE
Scatter with markers options

### DISCUSSION THREAD

**Discussion by kaleming** [kernc/backtesting.py#474] [Q&A] [answered]
Hi,

I would like to know if it is possible to define which type of marker certain indicator will display when scatter option is `True` ?

Something like the following:

```python

x1 = self.I(..., overlay=True, scatter=True, marker='triangle')
x2 = self.I(..., overlay=True, scatter=True, marker='circle')
```

**Comment by kernc** [comment, marked as answer]
You can specify _color_: [`self.I(..., color='pink')`](https://kernc.github.io/backtesting.py/doc/backtesting/backtesting.html#backtesting.backtesting.Strategy.I).

Must admit I'm not too hyped for adding styling parameters such as your markers or [additional plot styles](https://github.com/kernc/backtesting.py/issues/195). It deters from the underlying market science, which is the important part, and backtesting.py could never strive to replace TradingView.

If you can't color-code your indicators/markers according to your reason, you may already be plotting too many of them. :wink:

**Reply by kaleming** [reply]
Thanks for your reply @kernc . 
I was just curious to know if it was already implemented. I agree with you that it not essential.

Best regards
